src/script/descriptor.cpp

```diff
@@ -1556,7 +1600,9 @@ std::vector<std::unique_ptr<PubkeyProvider>> ParsePubkey(uint32_t key_exp_index,
     static_assert(sizeof(info.fingerprint) == 4, "Fingerprint must be 4 bytes");
     assert(fpr_bytes.size() == 4);
     std::copy(fpr_bytes.begin(), fpr_bytes.end(), info.fingerprint);
-    if (!ParseKeyPath(slash_split, info.path, apostrophe, error)) return {};
+    std::vector<KeyPath> path;
+    if (!ParseKeyPath(slash_split, path, apostrophe, error)) return {};
+    info.path = path.at(0);
```

Comment:
In fa53d191a02:

Could be explicit here. The general `slash_split` variable naming doesn't really help. 
```suggestion
    // Extract origin path elements
    std::vector<KeyPath> path;
    if (!ParseKeyPath(slash_split, path, apostrophe, error, /**allow_multipath=**/false)) return {};
    info.path = path.at(0); // multipath specifier disabled for origin information
```
.
Can also remove the `allow_multipath` default value from the `ParseKeyPath` signature.

Reply:
Removed the default value. The variable names weren't added/changed in this PR.